esm/html/document.js

```diff
@@ -77,13 +77,13 @@ export class HTMLDocument extends Document {
    */
   get title() {
     const {head} = this;
-    let title = head.getElementsByTagName('title').shift();
+    let title = head.getElementsByTagName('title')[0];
```

Comment:
I think these days I would go for `.at(0`) instead ... so this could be:

```js
get title() {
  const {head} = this;
  return head.getElementsByTagName('title').at(0)?.textContent || '';
}
```